php/blocks/class-loader.php

```diff
@@ -93,6 +93,16 @@ public function editor_assets() {
 			[ 'wp-blocks' ],
 			filemtime( $this->assets['path']['editor_style'] )
 		);
+
+		// Enqueue 'material-icons'.
+		if ( ! wp_style_is( 'material-icons', 'enqueued' ) ) {
+			wp_enqueue_style(
+				'material-icons',
+				'https://fonts.googleapis.com/icon?family=Material+Icons',
+				[],
+				'20180904'
```

Comment:
@rheinardkorf Is this the best date to use? Should we be including the material icons with the plugin, rather than loading them remotely?

Reply:
Its the date we decided to add it 😄  I have mixed feelings about adding it to the plugin or via CDN.  Getting it from CDN means all the styles are loaded too.  Doing it locally means a few more enqueues.

Reply:
We could do it via... https://google.github.io/material-design-icons/#setup-method-2-self-hosting